RubberduckTests/Refactoring/EncapsulateFieldTests.cs

```diff
@@ -926,12 +968,16 @@ public void Presenter_Reject_ReturnsNull()
             Assert.AreEqual(null, presenter.Show());
         }
 
-        //NOTE: The tests below are commented out pending some sort of refactoring that enables them
+        //TODO: The tests below are commented out pending some sort of refactoring that enables them
         //to actually *test* something.  Currently, all of the behavior the tests are looking for is
         //being mocked.
+        // SEE: https://github.com/rubberduck-vba/Rubberduck/issues/3072
 
-        /*
```

Comment:
pretty sure, yeap. I still want to keep that specific change and will amend the PR later.